Release 0.1.22 (#94)
PEtab-dev/libpetab-python: CHANGELOG.md

## 0.1 series

### 0.1.22

* Allow zero bounds for log parameters by @FFroehlich in
  https://github.com/PEtab-dev/libpetab-python/pull/83

> * Adapt to matplotlib 3.5 by @dweindl in

Reviewer: ```suggestion
* Adapt to Matplotlib 3.5 by @dweindl in
```

  https://github.com/PEtab-dev/libpetab-python/pull/86
* Allow specifying file format for visualization by @dweindl in
  https://github.com/PEtab-dev/libpetab-python/pull/85
* Visualization: Don't mess with rcParams by @dweindl in
  https://github.com/PEtab-dev/libpetab-python/pull/90
* Linter: Check condition IDs are unique by @dweindl in
  https://github.com/PEtab-dev/libpetab-python/pull/92

> * Add support for pathlib for reading PEtab tables by @dweindl, @dilpath

Reviewer: ```suggestion
* Add support for `pathlib` for reading PEtab tables by @dweindl, @dilpath
```

  in https://github.com/PEtab-dev/libpetab-python/pull/93,
  https://github.com/PEtab-dev/libpetab-python/pull/91

> * Run tests also on Python3.10 by @dweindl in

Reviewer: ```suggestion
* Run tests also on Python 3.10 by @dweindl in
```
